lib/commands.coffee

```diff
@@ -32,11 +32,11 @@ normalizeDrive = (drive) ->
 			throw new Error('Drive is not a string, nor an object with `raw` and `size` properties')
 
 
-		driveListAsync().then (drives) ->
-			return _.find(drives, device: drive)
-		.tap (foundDrive) ->
```

Comment:
Updated drivelist returns a native promise so `.tap` won't work, but since `_.find` is synchronous anyway I got rid of the need for it